### ISSUE TITLE
False positives on multiple sites.

### ISSUE THREAD

**Issue by hoerenraph** [thelinuxchoice/userrecon#7]
Running the script on random 16-digit names returns 33 false positives.
Tested with 3 different names, false positives were found on always the same 33 sites.
TXTs are attached. (file names are tested usernames)
[Knne2fygHMXNtSkG.txt](https://github.com/thelinuxchoice/userrecon/files/2721850/Knne2fygHMXNtSkG.txt)
[SnHbJN68Luj7YLBY.txt](https://github.com/thelinuxchoice/userrecon/files/2721851/SnHbJN68Luj7YLBY.txt)
[U94ZBQgeZM6RE5hu.txt](https://github.com/thelinuxchoice/userrecon/files/2721852/U94ZBQgeZM6RE5hu.txt)

**Comment by Z33DD** [comment]
Yes! I find the same problems
A solution is make a script to compare the error's page of this sites with the URL page.

**Comment by lonewolf9277** [comment]
It gives an okcupid profile for every username entered lol

**Comment by cryptax** [comment]
Yep. Same problem, that's because several social network create placeholder pages for any username...
